openid-4-verifiable-presentations-1_0.md

```diff
@@ -1894,6 +2048,7 @@ The technology described in this specification was made available from contribut
 
    * added references to ISO/IEC 23220 and 18013 documents 
    * added `post` request method for Request URI
+   * Added IETF SD-JWT VC profile
```

Comment:
wait... what? 
isn't this PR "adding a OID4VP profile for the W3C Digital Credentials API" or something along those lines?

Reply:
This is gitlab being screwy. It's showing a whole load of changes that aren't actually in this PR. (I checked by checking out locally, merging main into this branch and then doing a diff against main, and this change doesn't show up.)

I suggest merging main into this branch, that'll probably "fix" it.

Reply:
yes, this has been fixed.